detd/devices/intel_i210.py

```diff
@@ -23,6 +28,12 @@
 
 class IntelI210(Device):
 
+
+    NUM_TX_QUEUES = 4
+    NUM_RX_QUEUES = 4
+
+    CAPABILITIES  = [Capability.Qbv]
```

Comment:
i210 does not have the Qbv hardware capability. It does have Launch Time Control support.